pkgs/tombi-toml/tombi/registry.yaml

```diff
@@ -4,109 +4,44 @@ packages:
       - version_constraint: Version == "v0.3.27"
         no_asset: true
-      - version_constraint: Version == "v0.3.35"
-        asset: tombi-cli-{{trimV .Version}}-{{.Arch}}-{{.OS}}.{{.Format}}
-        format: gz
-        replacements:
-          amd64: x86_64
-          arm64: aarch64
-          darwin: apple-darwin
-          linux: unknown-linux-gnu
-          windows: pc-windows-msvc
-        overrides:
-          - goos: windows
-            format: zip
-            replacements:
-              amd64: amd64
-        supported_envs:
-          - linux
-          - darwin
-          - windows/arm64
-      - version_constraint: Version == "v0.3.36"
-        asset: tombi-cli-{{trimV .Version}}-{{.Arch}}-{{.OS}}.{{.Format}}
-        format: gz
-        replacements:
-          amd64: x86_64
-          arm64: aarch64
-          darwin: apple-darwin
-          linux: unknown-linux-gnu
-        supported_envs:
-          - linux
-          - darwin
-      - version_constraint: Version == "v0.3.37"
+      - version_constraint: semver("<= 0.2.23")
+        no_asset: true
+      - version_constraint: semver("<= 0.3.37")
         asset: tombi-cli-{{trimV .Version}}-{{.Arch}}-{{.OS}}.{{.Format}}
         format: gz
         replacements:
           amd64: x86_64
           arm64: aarch64
           darwin: apple-darwin
-          windows: pc-windows-msvc
-        overrides:
-          - goos: linux
-            goarch: amd64
-            replacements:
-              linux: unknown-linux-musl
-          - goos: linux
-            goarch: arm64
-            replacements:
-              linux: unknown-linux-gnu
-          - goos: windows
-            format: zip
-            replacements:
-              amd64: amd64
         supported_envs:
-          - linux
           - darwin
-          - windows/arm64
-      - version_constraint: Version == "v0.3.38"
+      - version_constraint: semver("<= 0.3.38")
         asset: tombi-cli-{{trimV .Version}}-{{.Arch}}-{{.OS}}.{{.Format}}
         format: gz
         windows_arm_emulation: true
         replacements:
           amd64: x86_64
           arm64: aarch64
           darwin: apple-darwin
+          linux: unknown-linux-musl
           windows: pc-windows-msvc
         overrides:
-          - goos: linux
-            goarch: amd64
-            replacements:
-              linux: unknown-linux-musl
-          - goos: linux
-            goarch: arm64
-            replacements:
-              linux: unknown-linux-gnu
           - goos: windows
             format: zip
-      - version_constraint: semver("<= 0.2.23")
-        no_asset: true
-      - version_constraint: semver("<= 0.3.8")
-        asset: tombi-cli-{{trimV .Version}}-{{.Arch}}-{{.OS}}.{{.Format}}
-        format: gz
-        replacements:
-          amd64: x86_64
-          arm64: aarch64
-          darwin: apple-darwin
-          linux: unknown-linux-gnu
+            files:
+              - name: tombi
+                src: "{{.FileName}}"
```

Comment:
The default value of `src` is `{{.FileName}}`, so this `src` is unnecessary.

Reply:
🙇‍♂️